src/sentry/integrations/vsts/integration.py

```diff
@@ -370,15 +384,16 @@ def get_pipeline_views(self):
 
         return [identity_pipeline_view, AccountConfigView()]
 
-    def build_integration(self, state):
+    def build_integration(self, state: Mapping[str, Any]) -> Mapping[str, Any]:
         data = state["identity"]["data"]
         oauth_data = self.get_oauth_data(data)
         account = state["account"]
         user = get_user_info(data["access_token"])
         scopes = sorted(self.get_scopes())
-        base_url = self.get_base_url(data["access_token"], account["accountId"])
+        # TODO(mgaeta): get_base_url() can return None.
```

Comment:
Shouldn't we handle that case?

Reply:
This a tough one for me. `base_url` is used in a lot of places that just won't work with `None`. I think it's a bug in `get_base_url()` that `None` is a possible return value. I don't want to fix the bug in this PR so I'll propagate the optionality of the string all the way down and leave a TODO.